Make it clear this does not work on Windows

![image](https://user-images.githubusercontent.com/475132/81373389-87ca1380-90b1-11ea-8a72-e968a81d665d.png)

Hi team, I built this awesome workflow around scp-action, ran it, then cried when I saw
```
##[error]Container action is only supported on Linux
```

I don't see mention of this limitation in any of the documentation.

--- a/README.md
+++ b/README.md
@@ -6,6 +6,8 @@
 
 [![Actions Status](https://github.com/appleboy/scp-action/workflows/scp%20files/badge.svg)](https://github.com/appleboy/scp-action/actions)
 
+**Importment**: Only support **Linux** container.
+
 ## Usage
 
 copy files and artifacts via SSH as blow.

